ietf-optical-impairment-topology.yang

```diff
@@ -952,9 +1147,178 @@ module ietf-optical-impairment-topology {
       "This augment is only valid for optical impairment with sliceable
       transponder model";
    }
-   description 
-     "Tunnel termination point augmentation for sliceable transponder
-     model.";
-   uses sliceable-transponder-attributes; 
+   description
+     "Tunnel termination point augmentation for sliceable transponder model.";
+   uses sliceable-transponder-attributes;
   }
+
+  augment "/nw:networks/nw:network/nw:node/tet:te"
+        + "/tet:te-node-attributes" {
+    when "/nw:networks/nw:network/nw:network-types"
+       + "/tet:te-topology/optical-imp-topo:optical-impairment-topology" {
+
+      description
+        "This augment is only valid for Optical Impairment topology";
+    }
+    description
+      "node attributes augmentantion for optical-impairment RAODM node";
+
+    list path-impairment {

```

Comment:
change to roadm-path-impairment ?

Reply:
I originally put exactly this name, but I received one comment realted to the fact,, in the choice you already have the mention of "roadm" (roadm-express-path, roadm-add roadm.drop) so no need still to highlight roadm word.